src/test/java/org/jabref/logic/importer/fetcher/CollectionOfComputerScienceBibliographiesFetcherTest.java

```diff
@@ -0,0 +1,116 @@
+        fetcher = new CollectionOfComputerScienceBibliographiesFetcher(importFormatPreferences);
+    }
+
+    @Test
+    public void getNameReturnsCorrectName() {
+        assertEquals("Collection of Computer Science Bibliographies", fetcher.getName());
+    }
+
+    @Test
+    public void getUrlForQueryReturnsCorrectUrl() throws MalformedURLException, URISyntaxException, FetcherException {
+        String query = "java jdk";
+        URL url = fetcher.getURLForQuery(query);
+        assertEquals("http://liinwww.ira.uka.de/bibliography/rss?query=java+jdk&sort=score", url.toString());
+    }
+
+    @Test
+    public void performSearchReturnsMatchingMultipleEntries() throws FetcherException {
+        List<BibEntry> searchResult = fetcher.performSearch("jabref");
+
+        BibEntry firstBibEntry = new BibEntry(StandardEntryType.InProceedings)
+                .withCiteKey("conf/ecsa/OlssonEW17")
+                .withField(StandardField.AUTHOR, "Tobias Olsson and Morgan Ericsson and Anna Wingkvist")
+                .withField(StandardField.EDITOR, "Rog{\\~A}{\\copyright}rio de Lemos")
+                .withField(StandardField.ISBN, "978-1-4503-5217-8")
+                .withField(StandardField.PAGES, "152--158")
+                .withField(StandardField.PUBLISHER, "ACM")
+                .withField(StandardField.TITLE, "The relationship of code churn and architectural violations in the open source software JabRef")
+                .withField(StandardField.URL, "http://dl.acm.org/citation.cfm?id=3129790")
+                .withField(StandardField.YEAR, "2017")
+                .withField(StandardField.BOOKTITLE, "11th European Conference on Software Architecture, ECSA 2017, Companion Proceedings, Canterbury, United Kingdom, September 11-15, 2017")
+                .withField(new UnknownField("bibsource"), "DBLP, http://dblp.uni-trier.de/https://doi.org/10.1145/3129790.3129810; DBLP, http://dblp.uni-trier.de/db/conf/ecsa/ecsa2017c.html#OlssonEW17")
+                .withField(new UnknownField("bibdate"), "2018-11-06");
+
+        BibEntry secondBibEntry = new BibEntry(StandardEntryType.Article)
+                .withCiteKey("oai:DiVA.org:lnu-68408")
+                .withField(new UnknownField("identifier"), "urn:isbn:978-1-4503-5217-8; doi:10.1145/3129790.3129810; ISI:000426556400034")
+                .withField(new UnknownField("subject"), "Software Architecture; Code Churn; Open Source; Architecrual Erosion; Technical Debt; Software Engineering; Programvaruteknik")
+                .withField(new UnknownField("relation"), "ACM International Conference Proceeding Series; ECSA '17~Proceedings of the 11th European Conference on Software Architecture : Companion Proceedings, p. 152-158")
+                .withField(StandardField.ABSTRACT, "The open source application JabRef has existed since\r\n" +
+                        "\t\t 2003. In 2015, the developers decided to make an\r\n" +
```

Comment:
Would it be possible to remove `\t\t ` from the abstract? 
Can we replace `\r\n` by the configured newline character (`OS.NEWLINE`)? --> Use `org.jabref.logic.formatter.bibtexfields.NormalizeNewlinesFormatter` in a CleanupJob.

`org.jabref.logic.formatter.bibtexfields.TrimWhitespaceFormatter` cannot be used for the first requriement. Think we need a new `TrimWhitespaceAndFixNewLinesInTextFieldFormatter`. Would it be possible for you to add such a Formatter?

Reply:
I can see that there is also an `RemoveNewlinesFormatter` - what is the reason for choosing to normalize the new lines rather than remove them? It looks like the new lines were added due for formatting reasons, and were not actually present in the original paper's abstract.

I therefore propose that I use the existing `RemoveNewlinesFormatter`, and also add a `RemoveTabFormatter` which removes any instance of `\t`.

What do you think?

Reply:
Yeah, let's see how it works out!